pkg/interface/src/views/apps/chat/components/lib/profile-overlay.js

```diff
@@ -85,7 +87,7 @@ export class ProfileOverlay extends PureComponent {
           <div className="mono gray2">{cite(`~${ship}`)}</div>
           {!isOwn && (
             <Link
-              to={`/~chat/new/dm/~${ship}`}
+              to={`/~todo/~${ship}`}
```

Comment:
TODO

Reply:
I'd suggest `/~groups/home/new/dm/<patp>`

Reply:
Performed the logic inline, seems neater, since this is the only place we can create a new DM.